cmd/sos_remove.go

```diff
@@ -10,9 +10,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// removeCmd represents the remove command
 var removeCmd = &cobra.Command{
-	Use:     "remove <bucket name> [object name]+",
+	Use:     "remove BUCKET [OBJECT-NAME]...",
```

Comment:
For consistency with other commands:
```suggestion
	Use:     "remove BUCKET [OBJECT]...",
```